spec/services/gtfs_graph_spec.rb

```diff
@@ -56,8 +56,8 @@ def load_feed(import_level=1)
       expect(r.geometry).to be
     end
 
-    it 'created known Stops' do
-      expect(@feed.imported_stops.count).to eq(31)
+    it 'created known Stops', :test_stops => true do
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.